docs/about/release-notes.md

```diff
@@ -51,7 +51,7 @@ details about these specific themes.
 [MkDocs Bootstrap]: http://mkdocs.github.io/mkdocs-bootstrap/
 [MkDocs Bootswatch]: http://mkdocs.github.io/mkdocs-bootswatch/
 
-They will be included with MkDocs by default until the 1.0 release. After that
+They will be included with MkDocs by default until a future release. After that
```

Comment:
This is the bad change, it's a slight change in contract. However, given that `1.0` is somewhat mythical and hasn't been defined I think it is okay.